go-controller/hybrid-overlay/pkg/controller/node_linux.go

```diff
@@ -53,26 +54,62 @@ func podToCookie(pod *kapi.Pod) string {
 	return nameToCookie(pod.Namespace + "_" + pod.Name)
 }
 
-func (n *NodeController) addOrUpdatePod(pod *kapi.Pod) error {
+func (n *NodeController) watchForNamespace(name string, wf *factory.WatchFactory) (*kapi.Namespace, error) {
```

Comment:
@rcarrillocruz can you rename `waitForNamespace()`?

Reply:
oh erm, that was my intent, bogus name sorry